### ISSUE TITLE
Touch to Search not working

### ISSUE THREAD

**Issue by rahaaatul** [kiwibrowser/src.next#552]
**Describe the bug**
Touch to search doesn't work. Even when I enable it, it shows it is off.

![Gif_20220325111128.gif](https://user-images.githubusercontent.com/86149861/160059024-0b6b78d5-d489-4782-82bf-4f2d3bbd6a07.gif)

**To Reproduce**
- Enable Touch to search in Google Services
- Touch any word

**Expected behavior**
It should display definitions, pictures, search results, and other details when I touch a word.

**Screenshots**

**Device Information**
 - Device: Vivo Y20
 - Android Version : A11
 - Kiwi Browser Version : 99.0.4844.86

**Comment by netrunner-exe** [comment]
All works. Xiaomi Mi 9 Lite android 11.
![IMG_20220328_134415](https://user-images.githubusercontent.com/81887288/160384825-94e047f6-eb2e-43c5-8207-69ee8fc72dfd.jpg)
![ezgif-1-a8a6bc786a](https://user-images.githubusercontent.com/81887288/160384976-773dc99d-f520-480f-ad57-c6e6a1b916de.gif)

**Comment by netrunner-exe** [comment]
Looks like only the displayed checkbox on / off in options is not working properly

**Comment by rahaaatul** [comment]
> Looks like only the displayed checkbox on / off in options is not working properly

Yes, it's on but says off. And also, it REALLY IS not working on my phone. How did you made the gifs? I want to show you mine too.

**Comment by rahaaatul** [comment]
https://ibb.co/HDRPRgY][img]https://i.ibb.co/HDRPRgY/20220328-222630.gif

**Comment by rahaaatul** [comment]
[img]https://ibb.co/HDRPRgY][img]https://i.ibb.co/HDRPRgY/20220328-222630.gif[/img]

**Comment by rahaaatul** [comment]
> Looks like only the displayed checkbox on / off in options is not working properly

Yes, it's on but displays it's off.
And also, it REALLY IS not working on my phone. 👇

![GIF-220328_230709.gif](https://user-images.githubusercontent.com/86149861/160451832-ff5664ca-1f7d-47be-afb6-b05b3dd1ab2e.gif)

_What app do you use to create gifs?_
